
var list = new List&lt;int&gt;{10, 42};
Assert.IsTrue(list.Remove(42)); // 削除失敗するとLogExceptionされる...?

